sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/metadata/column/ShardingGeneratedKeyColumnMetaData.java

```diff
@@ -31,7 +31,8 @@
 @ToString(callSuper = true)
 public final class ShardingGeneratedKeyColumnMetaData extends ColumnMetaData {
     
-    public ShardingGeneratedKeyColumnMetaData(final String name, final String dataType, final boolean primaryKey) {
-        super(name, dataType, primaryKey);
+    public ShardingGeneratedKeyColumnMetaData(final String name, final String dataType, final boolean primaryKey, 
+                                              final boolean nullable, final boolean autoIncrement) {
```

Comment:
Same as above.

Reply:
I hava already fixed it. Unified name is `notNull`.

Reply:
still nullable